contents/docs/integrate/_snippets/nextjs/pages-router-setup.mdx

```diff
@@ -55,23 +45,21 @@ export default function App({ Component, pageProps }) {
   useEffect(() => {
     posthog.init(process.env.NEXT_PUBLIC_POSTHOG_KEY, {
       api_host: process.env.NEXT_PUBLIC_POSTHOG_HOST || '<ph_client_api_host>',
+      capture_pageview: 'history_change',
       // Enable debug mode in development
       loaded: (posthog) => {
         if (process.env.NODE_ENV === 'development') posthog.debug()
       }
     })
 
-    const handleRouteChange = () => posthog?.capture('$pageview')
 
     const handleRouteChangeStart = () => posthog?.capture('$pageleave', {
       $current_url: oldUrlRef.current
     })
```

Comment:
You dont need `$pageleave` captures everytime you change pages, we handle that automatically via `$pageview`s in the backend. This can be removed.

Reply:
aww, awesome TIL